shared/js/dialer/keypad.js

```diff
@@ -635,7 +635,7 @@ var KeypadManager = {
             reject();
           };
 
-          ConfirmDialog.show('loadingContacts', null, {
+          ConfirmDialog.show('loadingSimContacts', null, {
             title: 'cancel',
```

Comment:
This should be tested in ```apps/communications/dialer/test/unit/keypad_test.js```. It should be a fairly trivial change, just to make it more specific about what's being passed in.